client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -33,7 +35,22 @@ class ActivityLogDay extends Component {
 			tsEndOfSiteDay,
 			requestRestore,
 		} = this.props;
-		requestRestore( tsEndOfSiteDay );
+		requestRestore( tsEndOfSiteDay, 'day' );
+	};
+
+	trackOpenDay = () => {
+		const {
+			logs,
+			moment,
+			recordTracksEvent,
+			tsEndOfSiteDay,
+		} = this.props;
+
+		recordTracksEvent( 'calypso_activity_log_day_expand', {
+			logCount: logs.length,
+			tsEndOfSiteDay,
+			utcDate: moment.utc( tsEndOfSiteDay ).format( 'YYYY-MM-DD' ),
```

Comment:
Good thinking. Maybe we should include the day's rewind point timestamp (utc last second of site-local day) as well as the utcDate. Does that sound like it would cover all the bases?